src/core/border_router/routing_manager.hpp

```diff
@@ -198,6 +198,11 @@ class RoutingManager : public InstanceLocator
 
     static constexpr uint8_t kOmrPrefixLength    = OT_IP6_PREFIX_BITSIZE; // The length of an OMR prefix. In bits.
     static constexpr uint8_t kOnLinkPrefixLength = OT_IP6_PREFIX_BITSIZE; // The length of an On-link prefix. In bits.
+    static constexpr uint8_t kBrUlaPrefixLength  = 48;                    // The length of a BR ULA prefix. In bits.
+    static constexpr uint8_t kNat64PrefixLength  = 96;                    // The length of a NAT64 prefix. In bits.
+
+    static constexpr uint8_t kOmrPrefixSubnetId   = 1; // The subnet ID of a OMR prefix within the BR ULA prefix site.
```

Comment:
```suggestion
    static constexpr uint16_t kOmrPrefixSubnetId   = 1; // The subnet ID of a OMR prefix within the BR ULA prefix site.
```
I think subnet ID is uint16_t by definition

Reply:
Updated.